src/Console/Command/FixCommand.php

```diff
@@ -184,7 +184,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $progressOutput = $showProgress && $stdErr
-            ? new ProcessOutput($stdErr, $this->eventDispatcher)
+            ? new ProcessOutput($stdErr, $this->eventDispatcher, count($resolver->getFinder()))
```

Comment:
To would enforce that the finders used must be rewindable for when after first iteration the limit is reached.